sema/check/__main__.py

```
@@ -0,0 +1,72 @@
+# your_submodule/__main__.py
+
+# your_submodule/cli.py
+import sys
```

Ruff

`sys` imported but unused

Remove unused import: `sys`

(F401)